### DISCUSSION TITLE
[Question][Hw4]

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#707] [Q&A]
![Screenshot from 2023-05-09 01-15-38](https://user-images.githubusercontent.com/89933766/236875821-25419704-4db9-4cf3-b5d3-4caa89bdaf03.png)
Can you explain me what is ( Object.t * command list ) in processes parameter of Acl.run? 
If possible, could you give an example of such an entry?

**Comment by tanapthetimid** [comment]
It is the list of commands inside an object. For example, in scenario 3: the "conferer" object (which itself is a subject when it starts executing) has 2 commands in its command list, "confer..." and "execute...".
